ios/FluentUI/Core/Theme/Tokens/TokenizedControlConfigurable.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// Internal protocol for all controls' internal `configuration` objects.
+protocol ControlConfiguration: NSObject, ObservableObject, Identifiable {
+    associatedtype TokenType: ControlTokens
+
+    /// Custom design token set for this control, to use in place of the control's default Fluent tokens.
+    var overrideTokens: TokenType? { get set }
+
+    /// Common token lookup method, which should contain a lookup via the current environment's `fluentTheme`.
+    var tokens: TokenType { get set }
+}
+
+/// SwiftUI-specific extension to `TokenizedControl` that adds information about `configuration`.
+protocol TokenizedControlConfigurable: TokenizedControlInternal {
```

Comment:
Maybe `ConfigurableTokenizedControl` might make more sense? I think Configurable being at the end makes it sound like a noun rather than an adjective. But then I think that also suggests that the other controls (UIKit) aren't configurable? Which isn't really the case. I guess a more boring but explicit name could be "TokenizedControlWithConfiguration", but I'm not a huge fan.

Reply:
🤔

Reply:
Going with `ConfigurableTokenizedControl`!